libsrc/osic1p/cputc.s

```diff
@@ -36,10 +41,13 @@ L1:     cmp     #$0D            ; LF?
 cputdirect:
```

Comment:
> Regarding the single user: Is he using CONIO features like cursor positioning and/or text attributes? If not then he might actually rather be a POSIX I/O user only driven to your CONIO because of POSIX I/O missing ;-)

That is actually the case, but conio is better than nothing.

> Regarding POSIX I/O based on CONIO: Other targets with well known firmware entry points for character output tend to have a redirection mechanism of one kind or another allowing BASIC programs to print to another terminal, a printer, a different video adapter, <...>. The idea of mapping POSIX I/O to those firmware entry points for character output is to allow those redirection mechanisms to work like with BASIC programs.

I will do some reseach, but as far as I know there are no such features in the OSI BASIC.